help/grandorgue.xml

```diff
@@ -9962,6 +9962,15 @@ release (or other attacks). 0 means automatic selection.
           </listitem>
         </varlistentry>
       </variablelist>
+      <varlistentry>
+          <term>Pipe999ReleaseTruncationLength</term>
+          <listitem>
+            <para>
+(int 0 - 10000, default: 0) Release truncation length in ms, to shorten the length of releases.
```

Comment:
1. This solution does not allow to disable the release at all. Maybe -1 is better?
2. How to set the release truncation on the Rank level? Maybe ReleaseTruncationLenght?

Reply:
Normally -1 is indicating a default, unchanged value.

Reply:
I'd withdrow my suggestion (1). All other related settings have the default value `0` that means not to limit the release length. So I agree with the current solution to have `0` by default. Otherwise we should change the old settings as well.